### DISCUSSION TITLE
flink tablec查询报 unexpected block data

### DISCUSSION THREAD

**Discussion by libra612** [apache/flink-cdc#1381] [Q&A] [answered]
flink-connector-mysql-cdc 版本是2.1.1,flink版本是1.13.3
sql-client.sh执行创建flink table脚本
然后查询table
select count(*) from ods_mf_staff;
报错[ERROR] Could not execute SQL statement. Reason:
java.io.StreamCorruptedException: unexpected block data
跟踪taskmanager和jobmanager日志，同步报同样的错误
根据搜索的解决方案：
flink-conf.yaml 添加classloader.resolve-order: parent-first
重启taskmanager和jobmanager，重新执行查询，错误任然存在

**Comment by Silengit** [comment, marked as answer]
Try to upgrade flink to 1.14.5(scala 2.11). I used 1.13.6(scala 2.12) and also encounted this problem.
